### ISSUE TITLE
"Mystery" objects must be prohibited

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#20]
Consider this code:

```
[] > foo
  bar 42 > x
```

Here, the `bar` doesn't have `+alias`, and it's not one of `org.eolang` objects (there are just about 20 of them there). Let's create a lint that would catch such a situation and complain about it (with an `error`).

**Comment by yegor256** [comment]
@maxonfjvipon WDYT?

**Comment by Marat-Tim** [comment]
@yegor256 How will we get a list of all the objects from `org.eolang`? I mean, it can change

**Comment by yegor256** [comment]
@h1alexbel can you answer this question? I remember, we've solved this problem earlier somewhere.

**Comment by Marat-Tim** [comment]
@yegor256 I don't think it's done yet

With `eo-parser 0.56.1` and `lints 0.0.47` I don't get warning in such program
```eo
+unlint comment-too-short
+unlint mandatory-home
+unlint mandatory-spdx
+unlint unused-void-attr
+unlint mandatory-version
+unlint mandatory-package
# No comments.
[start] > canonical
  takogoobjectatochonemozshetbit > bar
```

**Comment by yegor256** [comment]
@Marat-Tim I mean, we've solved the problem of finding the list of all EO "canonical" objects

**Comment by h1alexbel** [comment]
@yegor256 @Marat-Tim yes we did solve that problem. We have [LtReservedName](https://github.com/objectionary/lints/blob/master/src/main/java/org/eolang/lints/LtReservedName.java) for this. However, we didn't include it yet in `PkMono`. This ticket should complete it: https://github.com/objectionary/lints/issues/465
